modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -1123,24 +1122,41 @@ protected function render_tab_titles_html( $item_settings ): string {
 
 	protected function render_tab_containers_html( $item_settings ): string {
 		ob_start();
-		$this->print_child( $item_settings['index'] );
+		$this->print_child( $item_settings['index'], $item_settings );
 		$container_html = ob_get_clean();
 
-		$setting_key = $this->get_repeater_setting_key( 'tab_content', 'tabs', $item_settings['index'] );
-		$attribute_selector = '" data-id=';
-		$active_class = 0 === $item_settings['index'] ? ' e-active"' : '"';
+		$attribute_selector = 'class="';
+		$active_class = 0 === $item_settings['index'] ? $attribute_selector . 'e-active ' : $attribute_selector;
```

Comment:
I see that there are some cases that the class attr will contain no values E.g, `class=""`.
In those cases, do we still need to add an empty class attr?

Reply:
Updated. I have made the adding of classes conditional.